continuous_delivery_scripts/utils/git_helpers.py

```diff
@@ -401,7 +400,13 @@ def remote_branch_exists(self, branch_name: str) -> bool:
         return self.get_remote_branch(branch_name) is not None
 
     def _get_specific_changes(self, change_type: Optional[str], commit1: Any, commit2: Any) -> List[str]:
-        diff = commit1.diff(commit2)
+        diff = []
+        if not (commit1 or commit2):
+            return []
```

Comment:
could you just not have this return? if `diff` is empty then `diff_iterator` will be empty so `changes` will be empty which would be the same result as returning `[]` here.

and then just change the `else` to `elif commit2`

Reply:
👍